server.js

```diff
@@ -25,4 +27,8 @@ module.exports.rules = {
 
 	// It's a Webpack optimization, not necessary here
 	"import/dynamic-import-chunkname": "off",
+
+	// Disable test rules we don't like
+	"jest/no-export": "off",
+	"testing-library/render-result-naming-convention": "off",
```

Comment:
Why would we disable this rule? We've already fixed it in both App and Extension to be compliant iirc.

Reply:
I don't know, I just copied it from the extension. I can drop it from both places if it passes

Reply:
I see the problem, we disabled the rule for files that match these patterns: `["**/!(*.test)*.ts?(x)", "**/*.ts"]`, which are some of our test helpers and include render functions from libraries other than testing-library where using `rendered` or `wrapper` wouldn't be an anti-pattern. 

We want to keep the rule on "error` for actual `*.test.ts?(x)` files.